code/modules/projectiles/guns/misc/beam_rifle.dm

```diff
@@ -25,6 +25,7 @@
 	ammo_x_offset = 3
 	ammo_y_offset = 3
 	modifystate = FALSE
+	charge_sections = 1
```

Comment:
why are you touching this file

Reply:
Beam rifle gets an updated sprite also, this is needed so that it looks right.